test/test_torch.py

```diff
@@ -11366,9 +11402,20 @@ def test_bitwise_ops(self):
         x_clone ^= y
         self.assertEqual(x_clone, xor_result)
 
-    def test_invert(self):
-        x = torch.ByteTensor([0, 1, 1])
-        self.assertEqual((~x).tolist(), [1, 0, 0])
+    def test_op_invert(self):
+        res = 0xffff - torch.arange(127, dtype=torch.int8)
```

Comment:
what are you trying to test here?

Reply:
I see this was just taken from https://github.com/pytorch/pytorch/pull/22320/ so you can leave it.  It does look strange though --e.g. `0xffff - torch.arange(127, dtype=torch.int8)` might not even give you an int8 once we have type promotion.